src/serializers/extra.rs

```diff
@@ -424,8 +425,19 @@ impl CollectWarnings {
                 .get_type()
                 .qualname()
                 .unwrap_or_else(|_| PyString::new_bound(value.py(), "<unknown python object>"));
+
+            let input_str = safe_repr(value);
+            let mut value_str = String::with_capacity(100);
+            value_str.push_str("with value `");
+            match crate::errors::truncate_large_string(&mut value_str, input_str.to_cow()) {
+                Err(_) => value_str.clear(),
+                Ok(()) => {
+                    value_str.push_str("` ");
+                }
+            };
```

Comment:
what do you think of using if/else here?

```suggestion
            if crate::errors::truncate_large_string(&mut value_str, input_str.to_cow()).is_ok() {
                value_str.push_str("` ");
            } else {
                value_str.clear();
            }
```

Reply:
Or maybe your suggestion to `unwrap` is better.